CC/Sounder/receiver.cc

```diff
@@ -293,9 +293,13 @@ void Receiver::loopRecv(int tid, int core_id, SampleBuffer* rx_buffer)
             }
 
             size_t radio_idx = it - config_->n_bs_sdrs_agg().at(cell);
+            size_t num_packets = config_->reciprocal_calib()
+                    && radio_idx == config_->cal_ref_sdr_id()
+                ? 1
+                : num_channels; // receive only on one channel at the ref antenna
```

Comment:
num_packets = config_->reciprocal_calib() && 1    or      config_->reciprocal_calib() && num_channels.     Sets num_packets to a bool value?  Is this correct?